### PR TITLE
Fix version of pyzmq

### PR DESCRIPTION
**Pull request by neewy**
### Description of the Change

Yesterday, the team of ZMQ released their new version of wheel: https://pypi.org/project/pyzmq/#history 

Unfortunately it does not build, so I had to fix the version to the latest known working version, which is 17.1.2:

 
### Benefits

Now locust image can build!

### Possible Drawbacks 

None

### Usage Examples or Tests *[optional]*

`docker build -t iroha-locust .`

### CODE REVIEW

File: test/load/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM python:3.6.6-alpine3.8
 
 RUN apk --no-cache add g++ \
-      && pip install locustio pyzmq
+      && pip install locustio pyzmq==17.1.2
```
Comment (lebdron) [inline test/load/Dockerfile]:
```suggestion
      && pip install locustio pyzmq
```

File: test/load/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM python:3.6.6-alpine3.8
 
 RUN apk --no-cache add g++ \
```
Comment (lebdron) [inline test/load/Dockerfile]:
```suggestion
 RUN apk --no-cache add g++ zeromq-dev \
```

Comment (neewy) [reply test/load/Dockerfile]: 🆗

File: test/load/Dockerfile (hunk 1 above)
Comment (neewy) [reply test/load/Dockerfile]: 🆗

**Pull request merged**